0014/README.md

```diff
@@ -26,8 +31,15 @@ That seemed a little bit too loose and easy to make bad choices with so changed
 This makes sense to me.
 It seems like the service should not out live its parent, so we would want to terminate services at the end of a task if we were still under a task.
 
-The services would be running before any tasks are started, and will be killed at the end of the workflow.
-The service would thus need to be a client of the tinkerbell stream if it wanted to react to any state changes/events.
+#### Lifecycle
+
+The services will be started before any Tasks.
+All services will be required to be `Up` and `healthy` (if HEALTHCHECK is setup in the image) before any tasks are started.
+Services must be `Up` for the duration of the Workflow.
+Just like Actions, if a Service exits prematurely the Workflow will be considered Errored.
```

Comment:
@gianarb here we're going with what you said. I still kind of think maybe its ok to let the workflow continue on if a service that is no longer needed errors.

I intentionally worded it to have the services up the whole duration of the workflow. We could be ok with letting the service exit gracefully though too.

Reply:
I think you are right; I didn't have an opinion about that; I was just asking. 
The point you raised has a lot of sense; if an action requires a service that is unhealthy, the action will fail the workflow will follow. If a service is down, but nobody needs it, why should we mark the workflow as a failure? I think it has sense :) I was far from having a solid opinion about that.

Reply:
Cool lets do that then.